src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useEffect } from 'react';
-import { Button } from 'antd';
+import { Modal, Button } from 'antd';
```

Comment:
Used existing Modal component from antd to negate redundancy of creating nearly identical modal from scratch.